### PR TITLE
add support for GitHub Enterprise

### PR DESCRIPTION
**Pull request by mdb**
fixes issue #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ GH_CLIENT_ID=xyz123
 GH_CLIENT_SECRET=asdf123
 # The URL to redirect the user to once the authentication was successful
 REDIRECT_URL=https://google.com
+
```
Comment (mxstbr) [inline README.md]: nit: remove empty line

File: README.md [hunk 2]
```diff
@@ -22,6 +22,9 @@ GH_CLIENT_ID=xyz123
...
+# An optional GH_HOST if you're using GitHub Enterprise; defaults to `github.com`
```
Comment (mxstbr) [inline README.md]: nit: Let's change this to "Optional: Specify the GitHub host when using GitHub Enterprise"

File: index.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ const url = require('url')
 const querystring = require('querystring')
 const axios = require('axios')
 const { send } = require('micro')
+const ghURL = process.env.GH_HOST || 'github.com'
```
Comment (mxstbr) [inline index.js]: nit: Rename to `githubUrl`—we don't win anything for being concise here 😉

**Review by mxstbr — CHANGES_REQUESTED** [review]
This is great, didn't think of this use case! Three tiny nitpicks then we can ship this 🚢

File: README.md (hunk 1 above)
Comment (mdb) [reply README.md]: Will do. My logic in the separate line was to differentiate it from the required params. Line removal works for me!

File: README.md (hunk 2 above)
Comment (mdb) [reply README.md]: 👍

File: index.js (hunk 3 above)
Comment (mdb) [reply index.js]: Will do. My logic in calling it `githubURL` was to make it consistent in naming style used elsewhere, such as `createRedirectHTML`. `githubUrl` works for me.

**Review by mxstbr — APPROVED** [review]

**Comment by mxstbr** [comment]
Awesome, thanks!

**Pull request merged**